frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -66,6 +68,19 @@ interface Props {
   totalEntries: number;
 }
 
+function hasSymptoms(noSymptoms: boolean, symptoms: string) {
```

Comment:
This is the meat of the change really, would appreciate a review to make sure it makes sense. Also, some thought as to whether it makes sense to have this logic on the frontend or should the backend really know (and report) this.

Reply:
Your `additional information` section on the PR really spells out the problem here. The fact that we store a `noSymptoms` column in addition to the `symptoms` JSON bag is a little awkward. I think that `noSymptoms` is meant to be authoritative, and the contents of `symptoms` in the event of `noSymptoms: true` is unimportant.

Even if a user selects symptoms, _then_ selects "no symptoms," and submits the form, only `noSymptoms: true` is recorded, and the symptoms values themselves in the JSON bag are all falsey. I guess what I'm saying is that I don't know of a scenario where the "Unknown" fall-through state here can be reached.

If you do end up keeping this, you could use this functional chain + ternary in liue of that for loop, if you wanted:

```javascript
return Object.values(JSON.parse(symptoms)).includes("true") ? "Yes" : "Unknown"
```

Reply:
@nullflux the "Unknown" fall-through can technically happen because you _don't_ need to complete the test questionnaire prior to actually submitting a result